.gitignore

```diff
@@ -9,3 +9,6 @@
 coverage.txt
 profile.out
 /.glide
+
+# files generated during tests
+config/temp
```

Comment:
Rebase to the latest master and `rm -Rf config/temp nodepool/config/temp` and these uncommitted files won't appear again 💮 
If it appears, it is a bug. I'd rather like to notice bugs instead of ignoring them.